cmd/core-command/res/configuration.yaml

```diff
@@ -13,11 +13,20 @@ Service:
   Host: localhost
   Port: 59882
   StartupMsg: "This is the Core Command Microservice"
+  SecurityOptions:
+    Mode: ""
+    OpenZitiController: "openziti:1280"
+    OpenZitiServiceName: "edgex.core-command"
+
 Clients:
   core-metadata:
     Protocol: http
     Host: localhost
     Port: 59881
+    SecurityOptions:
+      Mode: "zerotrust"
+      OpenZitiController: "openziti:1280"
```

Comment:
Note this is the one place that these are needed since this is the only core service that needed this client, i.e. it isn't common. So the comose will need overrides on this service also.